.appveyor.yml

```diff
@@ -47,7 +50,6 @@ install:
     - ps: $env:PATH = "${env:PYTHON};${env:PYTHON}\Scripts;" + $env:PATH
     # deal with missing stdint.h as previously described
     # see https://github.com/swistakm/pyimgui/blob/master/.appveyor.yml
-    - ps: cp "C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\include\stdint.h" "C:\Users\appveyor\AppData\Local\Programs\Common\Microsoft\Visual C++ for Python\9.0\VC\include\stdint.h"
```

Comment:
Pretty sure this is an ancient trick for dealing with Python 2.7.  While MDAnalysis may delay the sunsetting for Python 2.7 a bit longer, we don't test 2.7 on Windows for unrelated reasons (we'd have to rewrite some of our C code) so this isn't needed here.